src/Sylius/Bundle/WebBundle/Resources/views/Backend/Order/macros.html.twig

```diff
@@ -97,10 +97,14 @@
                 {% include 'SyliusWebBundle:Backend/Order:_paymentState.html.twig' %}
             </td>
             <td style="width: 20%">
-                {{ order.customer.fullname }}<br>
-                <a href="{{ path('sylius_backend_customer_show', {'id': order.customer.id}) }}">
-                    {{ order.customer.email }}
-                </a>
+                {% if order.customer is not null %}
```

Comment:
Yes, this issue is caused by carts, which don't have assigned customer. So I fixed that problem in different way 👍